homeassistant/components/dyson/air_quality.py

```diff
@@ -0,0 +1,146 @@
+"""
+Support for Dyson Pure Cool Air Quality Sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/dyson/
+"""
+import logging
+
+from homeassistant.components.air_quality import (
+    AirQualityEntity, ATTR_ATTRIBUTION, ATTR_NO2, ATTR_PM_10, ATTR_PM_2_5)
+from homeassistant.components.air_quality import DOMAIN as AIQ_DOMAIN
+from homeassistant.components.dyson import DYSON_DEVICES, DOMAIN
+
+DEPENDENCIES = ['dyson']
+
+ATTRIBUTION = 'Dyson purifier air quality sensor'
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_VOC = 'volatile_organic_compounds'
+
+PROP_TO_ATTR = {
```

Comment:
Remove this. This is handled in the base air quality component. See below about state attributes.